conda-requirements.txt

```diff
@@ -2,7 +2,7 @@
 # conda create -n <name> --file conda-requirements.txt
 
 # Mandatory dependencies
-biggus>=0.12.0
+biggus>=0.13.0
```

Comment:
@ajdawson you need to make an equivalent change in `minimal_conda_requirements.txt` too. Given that the minimum requirements Travis run is not failing regardless, I am concerned that your test here is not doing what you want...